engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -496,10 +491,10 @@ case class JesBackend(actorSystem: ActorSystem)
     // Create the mappings. GLOB mappings require special treatment (i.e. stick everything matching the glob in a folder)
     wdlFileOutputs.distinct map { wdlFile =>
       val destination = wdlFile match {
-        case WdlSingleFile(filePath) => s"${backendCall.callGcsPath}/$filePath"
-        case WdlGlobFile(filePath) => backendCall.globOutputPath(filePath)
+        case WdlSingleFile(filePath) => jobDescriptor.callRootPath.resolve(filePath).toString
+        case WdlGlobFile(filePath) => JesBackend.globOutputPath(callRootPath(jobDescriptor), filePath).toString
```

Comment:
did you run anything with JES that would hit this function?  The code looks fine to me but it can be easy to introduce regressions in this area of code (something I know all too well :))